### PR TITLE
impl !Send for SpanGuard

### PR DESCRIPTION
**Pull request by That3Percent**

### CODE REVIEW

File: firestorm-enabled/src/internal.rs [hunk 1]
```diff
@@ -1,7 +1,34 @@
 use crate::*;
 pub use firestorm_core::{EventData, Start};
+use std::marker::PhantomData;
 
-pub struct SpanGuard;
+// See also 6cb371a0-d6f0-48be-87d2-8d824b82e0e7
```
Comment (Theodus) [inline firestorm-enabled/src/internal.rs:5]: What is this comment referencing? I assume it's related to negative trait bounds not being fully implemented.

**Review by Theodus — APPROVED** [review]
LGTM

File: firestorm-enabled/src/internal.rs (hunk 1 above)
Comment (That3Percent) [reply firestorm-enabled/src/internal.rs:5]: I do GUID "links" to make comments that reference each other bidirectionally. With this you can have sets of related comments.

Comment (That3Percent) [reply firestorm-enabled/src/internal.rs:5]: Oh yeah... In this case it's referring to duplicated code.

**Pull request merged**